Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:13.8-alpine3.11 as builder
+
+# Stage 1 - the build process
+WORKDIR /usr/src/app
+COPY ["package.json", "package-lock.json", "./"]
+RUN npm install
```

Comment:
Dependencies shouldn't be installed within the Docker container. Instead the packaged version should be copied here from the CI tasks result. 
A `npm install` run might change the result on every run, since dependencies can be overwritten on npmjs